StortSpelProjekt/Sandbox/src/main.cpp

```diff
@@ -191,6 +191,13 @@ int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE, LPSTR, int nCmdShow)
 #pragma endregion CreateScene1
 	char sceneName[10] = "scene0";
 	sceneManager->SetSceneToDraw(sceneManager->GetScene(sceneName));
+
+
+    // AUDIO TESTING, press SPACE to play, TAB to stop
+    Audio audiotest;
+    audiotest.OpenFile(engine.GetAudioEngine(), TEXT("../Vendor/Resources/Audio/AGameWithNoName.wav"));
```

Comment:
Testing is fine. engine.GetAudioEngine() should not be in main.cpp, instead create something like AudioEngine.CreateSound(string path) which returns a Audio.